src/screens/HomeScreen.tsx

```diff
@@ -77,7 +80,20 @@ const HomeScreen = () => {
       setCurrentGameState(getGameStateFromDate(new Date()));
     }, GameSettings.UPDATE_FREQUENCY); // check every second
     return () => clearInterval(intervalId);
-  });
+  }, [setCurrentGameState]);
```

Comment:
what does adding this dependency do

Reply:
useEffect needs a dependency array which wasn't there before, and setCurrentGameState (like other global state fucntions) is just a required dependency